app/controllers/mock.php

```diff
@@ -646,6 +649,69 @@
         $response->noContent();
     });
 
+App::get('/v1/mock/github/callback')
+    ->desc('Create installation document using GitHub installation id')
+    ->groups(['mock', 'api', 'vcs'])
+    ->label('scope', 'public')
+    ->param('installation_id', '', new Text(256, 0), 'GitHub installation ID', true)
+    ->param('projectId', '', new Text(2048), 'Project ID of the project where app is to be installed', true)
+    ->inject('gitHub')
+    ->inject('project')
+    ->inject('response')
+    ->inject('dbForConsole')
+    ->action(function (string $providerInstallationId, string $projectId, GitHub $github, Document $project, Response $response, Database $dbForConsole) {
+        if (empty($projectId)) {
+            $error = 'Installation requests from organisation members for the Appwrite GitHub App are currently unsupported. To proceed with the installation, login to the Appwrite Console and install the GitHub App.';
+            throw new Exception(Exception::GENERAL_ARGUMENT_INVALID, $error);
+        }
```

Comment:
To mark proejct ID required, you can remove last `true` param from `->param()` call.

```php
    ->param('projectId', '', new Text(2048), 'Project ID of the project where app is to be installed')
```

That way its required, so action wont even run if projected isn't provided.